modules/op-release-notes-1-14.adoc

```diff
@@ -326,3 +326,27 @@ With this update, {pipelines-title} General Availability (GA) 1.14.3 is availabl
 * Before this update, when you started many pipeline runs using {pac} at the same time and these pipelines runs included a `max-keep-run` annotation, the {pac} watcher was unable to reconcile the pipeline runs because of a race condition between deletion of existing pipeline runs and processing new pipeline runs. Because of this issue, some pipeline runs could not be processed.  With this update, the {pac} watcher processes pipeline runs.
 
 * Before this update, when you used the `tkn pr logs -f` command to view the logs for a running pipeline, the command line utility stopped responding, even if the pipeline run completed successfully. With  this update, the `tkn pr logs -f` command properly displays the log information and exits.
+
+[id="release-notes-1-14-4_{context}"]
+== Release notes for {pipelines-title} General Availability 1.14.4
+
+With this update, {pipelines-title} General Availability (GA) 1.14.4 is available on {OCP} 4.12 and later versions.
+
+[id="fixed-issues-1-14-4_{context}"]
+=== Fixed issues
+
+* Before this update, a large number of error messages referencing `tekton-pipelines-webhook.ConversionWebhook` was sometimes logged. With this update, unneeded conversion webhook configuration for the `ClusterTask` and `StepAction` Custom Resource Definitions (CRDs) was removed, and such error messages are no longer logged.
+
+* Before this update, some configuration settings using `options` sections in the `TektonConfig` custom resource (CR) did not work, because a race condition happened if the same setting was configured using both the `options` section and another field in the `TektonConfig` CR. With this update, the settings work.
+
+* Before this update, the {pipelines-shortname} console plugin pod did not move to the node specified using the `nodeSelector`, `tolerations`, and `priorityClassName` settings. With this update, the {pipelines-shortname} plugin pod moves to the correct node.
+
+* Before this update, some error messages were logged in the operator controller logs without the proper context information. With this update, error messages contain the required information.
+
+* Before this update, if the `pipelines-scc-rolebinding` rolebinding was missing or deleted in any namespace, the {pipelines-shortname} operator controller failed to create default resources in new namespaces correctly. With this update, the controller functions correctly.
```

Comment:
```suggestion
* Before this update, if the `pipelines-scc-rolebinding` rolebinding was missing or deleted in any namespace, the {pipelines-shortname} operator controller would fail to create default resources in new namespaces correctly. With this update, the controller functions correctly.
```